content/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file.md

```diff
@@ -249,6 +249,9 @@ Supported options
 {% endnote %}
 
 - `prefix` specifies a prefix for all commit messages.
+   When you specify a prefix for commit messages, {% data variables.product.prodname_dotcom %} will automatically add a colon between the defined prefix and the commit message, unless you end the end the prefix with a space.
+   The code snippet below provides examples of both in the same configuration file.
+
```

Comment:
> Can you add the following to that line, after prefix specifies a prefix for all commit messages.?
>
>When you specify a prefix for commit messages, {% data variables.product.prodname_dotcom %} will automatically add a colon between the defined prefix and the commit message, unless you end the end the prefix with a space. The code snippet below provides examples of both in the same configuration file.

I added this now, but I think this is false. Let's look at the implementation (https://github.com/dependabot/dependabot-core/blob/06702c83e5d964173504ddba6838c37fe37207ba/common/lib/dependabot/pull_request_creator/pr_name_prefixer.rb#L75-L83):

```
prefix += ":" if prefix.match?(/[A-Za-z0-9\)\]]\Z/)
```

So having whitespace at the end is one way of many ways to "disable" the colon. For example, if your prefix contains an umlaut (eg ö), then there also is no colon. 

Do you have an idea how to formulate this, so that this is more clear? Should we maybe add the regex to the documentation, so it's very clear when a colon is added?